RELEASE.md

```diff
@@ -0,0 +1,40 @@
+# Kabanero.io Website Release Process
+
+## Overview
+The content on the Kabanero.io Website is built from four different sources:
+- UI of the Website from: https://github.com/kabanero-io/kabanero-website
+- Docs from: https://github.com/kabanero-io/docs
+- Blogs from: https://github.com/kabanero-io/blogs
+- The guides from repositories in https://github.com/kabanero-io that begin with `guide-`
+
+## Website Release
+The website has its own independent release process.  Releases are versioned using standard semantic versioning of format **MAJOR.MINOR.PATCH**.
+
+
+## Guides Release
+There are no "releases" for the Kabanero.io Guides. As detailed in the [Kabanero.io Guides Contribution Guidelines](https://github.com/kabanero-io/draft-guides-template/blob/master/README.md), a repository containing a guide being drafted should begin with `draft-guide-`.  `draft-` ensures that the guide will not get published to the site.  Removing the `draft-` from the repo name so it begins with `guide-` ensures that our build process will pull in the guide during the next build.
```

Comment:
maybe add something to the second sentence like: "guides are an on/off situation. They are either published or not published depending on ...... "